content/operate/rc/security/access-control/access-management.md

```diff
@@ -121,7 +121,7 @@ This table shows each role's ability to perform common tasks.
 | Create database (affects cost) | <span title="Owners can create databases when doing so will impact costs">&#x2705; Yes</span> | <span title="Billing Admins may not create databases when doing so will impact costs">&#x274c; No</span> | <span title="Managers can create databases when doing so will impact costs">&#x2705; Yes</span> | <span title="Members may not create databases when doing so will impact costs">&#x274c; No</span> | <span title="Viewers may not create databases when doing so will impact costs">&#x274c; No</span> | <span title="Logs viewers may not create databases when doing so will impact costs">&#x274c; No</span> |
 | Edit database (affects cost) | <span title="Owners can edit databases in ways that will impact costs">&#x2705; Yes</span> | <span title="Billing Admins may not edit databases in ways that impact costs">&#x274c; No</span> | <span title="Managers can edit databases in ways that impact costs">&#x2705; Yes</span> | <span title="Members may not edit databases in ways that impact costs">&#x274c; No</span> | <span title="Viewers may not edit databases in ways that impact costs">&#x274c; No</span> | <span title="Logs viewers may not edit databases in ways that impact costs">&#x274c; No</span> |
-| Create database (no cost impact) | <span title="Owners can create databases when doing so will not impact costs">&#x2705; Yes</span> | <span title="Billing Admins may not create databases when doing so will not impact costs">&#x274c; No</span> | <span title="Managers can create databases when doing so will not impact costs">&#x2705; Yes</span> | <span title="Members can create databases when doing so will not impact costs">&#x2705; Yes</span> | <span title="Viewers may not create databases when doing so will not impact costs">&#x274c; No</span> | <span title="Logs viewers may not create databases when doing so will not impact costs">&#x274c; No</span> |
+| Create database (no cost impact) | <span title="Owners can create databases when doing so will not impact costs">&#x2705; Yes</span> | <span title="Billing Admins may not create databases when doing so will not impact costs">&#x274c; No</span> | <span title="Managers can create databases when doing so will not impact costs">&#x2705; Yes</span> | <span title="Members can not create databases when doing so will not impact costs">&#x274c; No</span> | <span title="Viewers may not create databases when doing so will not impact costs">&#x274c; No</span> | <span title="Logs viewers may not create databases when doing so will not impact costs">&#x274c; No</span> |
```

Comment:
This sentence just doesn't seem right to me. "**Members can not create databases when doing so will not impact costs**". Seems like there should be another reason why members can't create databases in this situation. Like, maybe members just can't create databases at all?

Reply:
Yes, but this is alt text for this row so all of the cells have similar alt text even if the role can't create databases at all.